test/lib/prisma.test.ts

```diff
@@ -0,0 +1,101 @@
+
+import {it, expect} from '@jest/globals';
+import {whereAndSelect} from "@lib/prisma";
+
+it('can decorate using whereAndSelect', async () => {
+  whereAndSelect( (queryObj) => {
```

Comment:
Codacy has a fix for the issue: [Insert `⏎···`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7713215)
```suggestion
  whereAndSelect(
```